Update all playwright dependencies to 1.17.2

unlikelyzero opened the pull request:
Updates Playwright dependencies because someone broke the internet

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [ ] Changes address original issue?
* [x] Unit tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [ ] Testing instructions included in associated issue?

trusktr commented:
After an npm install, we can grep for `colors@latest` to find any culprits.

(one should never use `@latest`!)

unlikelyzero commented:
@trusktr ok, pinned in devDependencies

unlikelyzero commented:
@trusktr it looks like this is the last PR with the colors dependency. I've removed the pinned colors and reverted it back to the original state (that you approved :) ). When you get a chance, can you re-approve?
